practiceQuestion2.rb

```diff
@@ -0,0 +1,18 @@
+no_of_element = gets().chomp().to_i
```

Comment:
don't use empty brackets.
```suggestion
no_of_element = gets.chomp.to_i
```